src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -89,9 +112,45 @@ def drop_application(self, auto_yes: bool):
                     cc.message(f"Did not drop application object {self.app_name}.")
                     return  # The user desires to keep the app, therefore exit gracefully
 
-        # 4. All validations have passed, drop object
+        # 4. Check for application objects owned by the application
+        application_objects = self.get_objects_owned_by_application()
+        if len(application_objects) > 0:
+            application_objects_str = self._application_objects_to_str(
+                application_objects
+            )
+            if cascade is True:
+                cc.message(
+                    f"The following objects are owned by application {self.app_name} and will be dropped:\n{application_objects_str}"
+                )
+            elif cascade is False:
+                cc.message(
+                    f"The following objects are owned by application {self.app_name}:\n{application_objects_str}"
+                )
+            elif interactive:
+                if interactive:
+                    api_integration = typer.prompt(
+                        f"The following objects are owned by application {self.app_name}:\n{application_objects_str}\n\nWould you like to drop these objects as well as the application? [y/n/ABORT]"
```

Comment:
```suggestion
                        f"The following objects are owned by application {self.app_name}:\n{application_objects_str}\n\nWould you like to drop these objects as well, or just the application? [y/n/ABORT]"
```

Reply:
Not sure about this wording, the answer is no longer yes/no this way

Reply:
My point was that "no" could either mean "drop neither" or "just drop the application". You're not being clear about which option maps to which outcome.

Reply:
What do you think of:

Would you like to drop these objects in addition to the application?